src/components/ProvideDetails.jsx

```diff
@@ -287,7 +290,8 @@ const provideDetails = props => {
 						<ButtonDisplay
```

Comment:
You can use the SeButton component here. It will handle adding the 'seButton' and do some other nice things for you if we need them. you will still need to add className="pull-right" to ensure its pulled right